### PR TITLE
Update copy

### PR DESCRIPTION
**Pull request by hannako**
- content changes
- reorder sections
- remove some links from the How you can help section

### CODE REVIEW

File: config/locales/en/coronavirus_landing_page.yml [hunk 1]
```diff
@@ -23,34 +22,6 @@ en:
                 url: /government/publications/covid-19-stay-at-home-guidance/stay-at-home-guidance-for-households-with-possible-coronavirus-covid-19-infection
```
Comment (sihugh) [inline config/locales/en/coronavirus_landing_page.yml]: Think this is just `/government/publications/covid-19-stay-at-home-guidance` now - so it's to the splash page, not the html attachment

File: config/locales/en/coronavirus_landing_page.yml [hunk 2]
```diff
@@ -23,34 +22,6 @@ en:
...
               - label: Find out what social distancing is and how to protect vulnerable people
                 url: /government/publications/covid-19-guidance-on-social-distancing-and-for-vulnerable-people/guidance-on-social-distancing-for-everyone-in-the-uk-and-protecting-older-people-and-vulnerable-adults
```
Comment (sihugh) [inline config/locales/en/coronavirus_landing_page.yml]: same here

File: config/locales/en/coronavirus_landing_page.yml [hunk 3]
```diff
@@ -101,24 +82,40 @@ en:
...
+                url: /guidance/coronavirus-covid-19-and-prisons
           - title: Driving and transport
             list:
               - label: Driving and theory tests
                 url: /guidance/coronavirus-covid-19-driving-tests-and-theory-tests
           - title: Education
             list:
-              - label: Schools, childcare, further and higher education
+              - label: "School closures: advice for parents and carers"
+              - url: /government/publications/closure-of-educational-settings-information-for-parents-and-carers
```
Comment (sihugh) [inline config/locales/en/coronavirus_landing_page.yml]: remove the leading `-`

Comment (hannako) [reply config/locales/en/coronavirus_landing_page.yml]: thank you

**Pull request merged**